src/hooks/use-aggrid-sort.ts

```diff
@@ -51,16 +49,16 @@ const getSortConfig = (
 
 interface IUseAgGridSortProps {
     dataKeyToSortKey?: DataKeyToSortKey;
-    initSortConfig?: ISortConfig;
+    initSortConfig: ISortConfig;
 }
 
 export const useAgGridSort = ({
     dataKeyToSortKey,
     initSortConfig,
-}: IUseAgGridSortProps = {}) => {
+}: IUseAgGridSortProps) => {
     const { colKey: initColKey, sortWay: initSortWay } = initSortConfig || {
         colKey: '',
```

Comment:
```suggestion
    const { colKey: initColKey, sortWay: initSortWay } = initSortConfig;
```

initSortConfig is mandatory, we don't need to define a default value

Reply:
done, thnak you